src/components/service.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+    font-family: 'Inter';
+    src: local('Inter'), url(../fonts/Inter/Inter-VariableFont_slnt,wght.ttf) format('truetype');
+    font-weight: 100 1000;
+	font-stretch: 25% 151%
+  }
+
+div{
+    font-family: Inter;
+    font-weight: 350;
+    text-align: left;
+}
+
+.heading{
+    margin: auto;
+    width: 50%;
+    padding: 10px;
+    text-align: center;
+    top: 100px;
+    color: rgb(255, 255, 255);
+}
+
+.textBackground {
+    padding: 20px;
+    position: relative;
+    max-width: 1000px;
+    width: 80%;
+    background: rgb(33, 36, 41);
+    box-shadow: rgb(0 0 0 / 1%) 0px 0px 1px, rgb(0 0 0 / 4%) 0px 4px 8px, rgb(0 0 0 / 4%) 0px 16px 24px, rgb(0 0 0 / 1%) 0px 24px 32px;
+    border-radius: 30px;
+    top: 3vh;
+    height: 27vh;
```

Comment:
You've given this a fixed height, which means that it won't stretch if the text is longer. Try removing these lines
```
position: relative
top: 3vh
height: 27vh
```
and move the button outside of the textbackground div